dashboard/app/controllers/registrations_controller.rb

```diff
@@ -295,11 +295,11 @@ def set_student_information
     student_information = {}
 
     student_information[:age] = params[:user][:age] if current_user.age.blank?
-    student_information[:us_state] = params[:user][:us_state] unless current_user.user_provided_us_state
+    us_state_param = params[:user][:us_state]
+    student_information[:us_state] = us_state_param if us_state_param && !current_user.user_provided_us_state
     student_information[:user_provided_us_state] = params[:user][:us_state].present? unless current_user.user_provided_us_state
```

Comment:
If the issue is that we are setting `us_state` to nil because there is no `current_user.user_provided_state`, would the same thing happen for `student_information[:user_provided_us_state]`?

Reply:
I think that property works because it doesn't get set at all, whereas us_state was getting set to the value in params, which was `nil`. `us_state` also has a special validation that makes it required, but `user_provided_us_state` doesn't.